### PR TITLE
fix: replaced old namings with adapted namings

### PR DESCRIPTION
**Pull request by antiyro**
just replaced:

- deoxys->madara
- Deoxys->Madara
- kasarlabs -> madara-alliance

and some details

### CODE REVIEW

**Review by apoorvsadana — APPROVED** [review]

File: scripts/deoxys
Comment (jbcaron) [inline scripts/deoxys:1]: rename this file

File: crates/client/sync/Cargo.toml [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-homepage = "https://github.com/KasarLabs/deoxys"
+homepage = "https://github.com/madara-alliance/madara"
 license = "MIT"
-name = "dc-sync"
+name = "mc-sync"
 publish = false
-repository = "https://github.com/KasarLabs/deoxys"
-version = "0.1.0"
+repository = "https://github.com/madara-alliance/madara"
+version = "0.7.0"
```
Comment (jbcaron) [inline crates/client/sync/Cargo.toml]: why this crate does not take the attributes of the workspace?

**Review by jbcaron — CHANGES_REQUESTED** [review]
In the future it would be nice to rename MadaraBackend, MadaraStorageError... more simply

File: crates/client/sync/Cargo.toml (hunk 1 above)
Comment (antiyro) [reply crates/client/sync/Cargo.toml]: changed it to match workspace attribute

File: scripts/deoxys
Comment (antiyro) [reply scripts/deoxys:1]: done

**Review by jbcaron — APPROVED** [review]

**Pull request merged**